contracts/bepro/Network_Registry.sol

```diff
@@ -90,12 +116,56 @@ contract Network_Registry is ReentrancyGuardOptimized, Governed {
 
     function changeAmountForNetworkCreation(uint256 newAmount) external payable onlyGovernor {
         require(newAmount > 0, "C1");
-        lockAmountForNetworkCreation = newAmount * 10 ** 18;
+        lockAmountForNetworkCreation = newAmount;
     }
 
     function changeLockPercentageFee(uint256 newAmount) external payable onlyGovernor {
         require(newAmount.div(10000) <= 10, "CLF1");
         lockFeePercentage = newAmount;
     }
 
+    function changeGlobalFees(uint256 _closeFee, uint256 _cancelFee) external payable onlyGovernor {
```

Comment:
Why dont we have a separate setter for each fee ? Space ???

Reply:
@hvasconcelos I didn't see a situation where anywould would want to change _only one_ since this values already exist on the constructor if someone is changing this they most likely want to change two values - so this would optimize that case. if the user simply wants to change _one_ then all it's needed is to _send the same value_